app/controllers/concerns/view_component/preview_actions.rb

```diff
@@ -23,6 +23,8 @@ def index
     end
 
     def previews
+      find_preview
```

Comment:
Just curious, why was this change necessary?

Reply:
Good question. Since we're calling `previews` directly from `render_preview`, `before_action` isn't called. In fact, I think we could inline `find_preview` here!